website/content/en/docs/upgrading-sdk-version/v1.6.0.md

```diff
@@ -50,7 +50,7 @@ _See [#4701](https://github.com/operator-framework/operator-sdk/pull/4701) for m
 
 ## (ansible/v1) Update Ansible collections
 
-In your requirements.yml, change the `version` field for community.kubernetes to `1.2.1`, and the `version` field for `operator_sdk.util` to `0.2.0`.
+In your requirements.yml, change the `version` field for kubernetes.core to `1.2.1`, and the `version` field for `operator_sdk.util` to `0.2.0`.
```

Comment:
2.2.0

Reply:
Actually you can revert the changes to the previous migration/upgrade docs. Since they are historical they are correct.